README.md

```diff
@@ -1 +1,56 @@
-# helloMicroverse
\ No newline at end of file
+# ![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello_Microverse
+
+> A simple project to show how linters, GitHub workflow and branches work
+
+
+## Built With
+
+- HTML
+- CSS
+
+
+
+
+## Getting Started
+
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- HTML5
+- CSS3
+- VSCode or any source code editor of your choice
+- GitHub
+- Terminal
+
+
+## Authors
+
+👤 **Okolo Oshoke**
+
+- GitHub: @lorenzoworx (https://github.com/lorenzoworx)
+- Twitter: @lorenzoworx(https://twitter.com/lorenzoworx)
+- LinkedIn: Oshoke Okolo(https://www.linkedin.com/in/oshoke-okolo-665208108)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page(https://github.com/ooshoke/Hello_Microverse/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Thanks to my coding partner Losalini for helping me make sense of this
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] As stated in  the project requirement kindly update your README file with new -  [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.